api/tests/routes/public/helpers.py

```diff
@@ -130,6 +132,34 @@ def setup_active_venue_provider(
         return plain_api_key, venue_provider
 
 
+class PublicAPIRestrictedEnvHelper(PublicAPIEndpointBaseHelper):
```

Comment:
Cool que tu aies rajouté ça 👍🏻 .

Dans le nom il manque un petit `Endpoint`
```suggestion
class PublicAPIRestrictedEnvEndpointHelper(PublicAPIEndpointBaseHelper):
```